src/webui/www/private/scripts/mocha-init.js

```diff
@@ -93,6 +93,20 @@ let copyCommentFN = function() {};
 let setQueuePositionFN = function() {};
 let exportTorrentFN = function() {};
 
+const baseModalOptions = Object.freeze({
```

Comment:
Maybe it could be moved into initializeWindows() to avoid polluting the global?

Reply:
There is at least a couple of windows outside initializeWindows that could be potentially made into modals.

Reply:
Then it can be wrapped into something like this: https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/www/private/scripts/cache.js#L31-L39

Reply:
Ah, I moved it there but you probably meant for me to create a similar thing in mocha-init? Or could it stay where it is now?

Reply:
>but you probably meant for me to create a similar thing in mocha-init? 

Yes.